src/components/templates/Jazzbar/components/JazzBarRoom/Room.tsx

```diff
@@ -126,20 +106,16 @@ const Room: React.FC<RoomProps> = ({
     setSeatedAtTable && setSeatedAtTable("");
   }, [firebase, profile, setSeatedAtTable, user, venueName]);
 
-  useEffect(() => {
-    if (!token) return;
-
-    let localRoom: Video.Room;
-
-    const participantConnected = (participant: Video.Participant) => {
-      setParticipants((prevParticipants) => [
-        // Hopefully prevents duplicate users in the participant list
-        ...prevParticipants.filter((p) => p.identity !== participant.identity),
-        participant,
-      ]);
-    };
+  const participantConnected = useCallback((participant: Video.Participant) => {
+    setParticipants((prevParticipants) => [
+      // Hopefully prevents duplicate users in the participant list
+      ...prevParticipants.filter((p) => p.identity !== participant.identity),
```

Comment:
This reduces the need to invent token variables

```suggestion
      ...prevParticipants.filter(({ identity }) => identity !== participant.identity),
```

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/2647/commits/e96d5d0b1b518b09b0c335ad0710dc76ac006101